app/css/lab-orders-list.scss

```diff
@@ -101,25 +101,6 @@ i.scale:hover {
   cursor: pointer;
 }
```

Comment:
Did some style tweaks... I wouldn't say the page looks great now, but at least in my opinion I think it looks better!